governance.md

```diff
@@ -153,49 +289,135 @@ are enumerated below.
 
 ## Other duty-specific github roles:
 - [**Github Org Owner**](https://github.com/orgs/kubernetes/people?utf8=%E2%9C%93&query=%20role%3Aowner):
-  can create repos, do ~any github action; the number of
+  can create repos, and do any github action; the number of
   owners shouldn't scale with the organization's growth, O(1), and optimally it
   should be less than 20 people who are very familiar with project workings and
   distributed across a few time zones and organizations The other repos will
   have distinct sets of people filling some of the above roles, also.
 
+## Procedural roles
+
+- **Champion**
```

Comment:
I would pivot this so that these are local to the process that they apply to.  We have a process for creating SIGs and for creating incubation projects and for adding people to the MAINTAINERS rank.  Each of those may require a champion of sponsor from some group.  But that doesn't mean that "Champion" is a role in any broad sense outside of those processes.

In fact, we may require a different "Champion" from different groups for each process.  It seems like you are trying to force a commonality here that doesn't exist.  It just ends up bloating this document, making it harder to read and obfuscates the processes themselves.